disttask/importinto/proto.go

```diff
@@ -32,16 +32,23 @@ import (
 // the initial step is StepInit(-1)
 // steps are processed in the following order:
 // - local sort: StepInit -> StepImport -> StepPostProcess -> StepDone
-// - global sort: StepInit -> StepEncodeAndSort -> StepWriteAndIngest -> StepPostProcess -> StepDone
+// - global sort:
+// StepInit -> StepEncodeAndSort -> StepMergeSort -> StepWriteAndIngest
+// -> StepPostProcess -> StepDone
 const (
 	// StepImport we sort source data and ingest it into TiKV in this step.
 	StepImport int64 = 1
 	// StepPostProcess we verify checksum and add index in this step.
 	StepPostProcess int64 = 2
 	// StepEncodeAndSort encode source data and write sorted kv into global storage.
 	StepEncodeAndSort int64 = 3
+	// StepMergeSort merge sorted kv from global storage, so we can have better
+	// read performance during StepWriteAndIngest.
+	// depends on how much kv files are overlapped, there's might 0 subtasks
+	// in this step.
+	StepMergeSort int64 = 4
```

Comment:
how about iota

Reply:
i plan to add a pr to put all steps together, so we can have a stepStr in framework code
we can change it then